source/store.go

```diff
@@ -354,3 +372,36 @@ func NewContourClient(kubeConfig, kubeMaster string, requestTimeout time.Duratio
 
 	return client, nil
 }
+
+func NewOpenShiftClient(kubeConfig, kubeMaster string, requestTimeout time.Duration) (*openshift.Clientset, error) {
```

Comment:
Same here please add documentation for this function.

Reply:
Copied and modified comment from ContourClient